ingest/ingest_pipeline.py

```diff
@@ -315,34 +318,31 @@ def ingest_expression(self) -> int:
                 )  # Dense receiver
             self.expression_ingestor.execute_ingest()
         except Exception as e:
-            self.error_logger.error(e, extra=self.extra_log_params)
+            IngestPipeline.dev_logger.critical(e, exc_info=True)
+            IngestPipeline.user_logger.critical(str(e))
             return 1
         return 0
 
     def ingest_cell_metadata(self):
         """Ingests cell metadata files into Firestore."""
         self.cell_metadata.preprocess()
         if self.cell_metadata.validate():
-            self.info_logger.info(
-                f"Cell metadata file format valid", extra=self.extra_log_params
-            )
+            IngestPipeline.dev_logger.error(f"Cell metadata file format valid")
             # Check file against metadata convention
             if self.kwargs["validate_convention"] is not None:
                 if self.kwargs["validate_convention"]:
                     if self.conforms_to_metadata_convention():
-                        self.info_logger.info(
-                            f"Cell metadata file conforms to metadata convention",
-                            extra=self.extra_log_params,
+                        IngestPipeline.dev_logger.info(
+                            f"Cell metadata file conforms to metadata convention"
                         )
                         pass
                     else:
                         return 1
 
             self.cell_metadata.reset_file()
             for metadataModel in self.cell_metadata.transform():
-                self.info_logger.info(
-                    f"Attempting to load cell metadata header : {metadataModel.annot_header}",
-                    extra=self.extra_log_params,
+                IngestPipeline.dev_logger.info(
+                    f"Attempting to load cell metadata header : {metadataModel.annot_header}"
```

Comment:
Since this ticket is for refactoring, please rename this variable here and elsewhere to use `snake_case`:

```suggestion
                    f"Attempting to load cell metadata header : {metadata_model.annot_header}"
```